src/test/ui/consts/issue-52432.rs

```diff
@@ -1,10 +0,0 @@
-#![feature(const_raw_ptr_to_usize_cast)]
-
-fn main() {
-    [(); &(static |x| {}) as *const _ as usize];
-    //~^ ERROR: closures cannot be static
-    //~| ERROR: type annotations needed
-    [(); &(static || {}) as *const _ as usize];
-    //~^ ERROR: closures cannot be static
-    //~| ERROR evaluation of constant value failed
```

Comment:
I think this has long stopped testing the right thing... looks like someone adjusted the expected errors without realizing that this makes the test pointless.

Reply:
Ah, looks like this was actually about testing an ICE: https://github.com/rust-lang/rust/pull/66331.
But with the feature gate gone I don't think keeping the test makes a lot of sense.